### ISSUE TITLE
Enum value of TRUE breaks schema 

### ISSUE THREAD

**Issue by rlancer** [jimkyndemeyer/js-graphql-intellij-plugin#244]
My schema has enums with the value "TRUE" in them, remove the "TRUE"s and everything works fine. 

Works
``` graphql
enum VideoDimensionSearchEnumParam {
  _2d
  _3d
  any
}
```
Breaks everything
``` graphql
enum VideoDimensionSearchEnumParam {
  _2d
  _3d
  any
  TRUE
}
```

**Comment by rlancer** [comment]
enum VideoDimensionSearchEnumParam {
  #Restrict search results to exclude 3D videos.
  _2d
  #Restrict search results to only include 3D videos.
  _3d
  #Include both 3D and non-3D videos in returned results. This is the default value.
  any
}

**Comment by malcolmwhite** [comment]
I'm running into this as well. FALSE also breaks the schema, unsurprisingly.

**Comment by kagof** [comment]
I'm guessing this is a result of https://github.com/jimkyndemeyer/js-graphql-intellij-plugin/blob/master/src/grammars/GraphQLParser.bnf#L103

The lowercase values are specified there (as per the GraphQL spec https://graphql.github.io/graphql-spec/June2018/#sec-Enum-Value, `true`, `false`, and `null` are invalid enum values).

However, according to https://tools.ietf.org/html/rfc7405, string literals are matched case insensitively.

**Comment by jimkyndemeyer** [comment]
Fixed in the upcoming 2.1.1 release.
